### PR TITLE
Papa 8 pre-cruise deploy csv updates

### PR DESCRIPTION
**Pull request by rtravis-whoi**
Pre-cruise updates for Papa 8 deploy csvs per mooring configs in Vault.  

WFP QCTs are not up on Vault yet but you can confirm WFP sensor SNs if you go to Platform_Records > WFPs > Global_WFPs and look in the folders for each WFP on HYPM.

NOTE: I added 'dummy' deploy date/time, lat/lon, nominal and deploy depths since it seemed to cause a few issues on the Pioneer deploy csv prep to have those cells blank.  Nominal depths on HYPM are slightly different due to design updates (stayed the same on FLMA/B) - refer to **CGSN_Mooring_Component_Depths.xlsx** file in Vault > Project_Files > Documentation > System > System Notebook > Asset Management

Also for HYPM, removed the ZPLSG lines from deployments 1-3 as part of 'clean-up' effort - those were not part of the design for the early deployments.

### CODE REVIEW

**Comment by rtravis-whoi** [comment]
[Papa 8 AM Updates [DT-65]](https://trello.com/c/NXHxU6Q7/65-papa-8-am-updates-dt-65)

File: deployment/GP03FLMB_Deploy.csv [hunk 1]
```diff
@@ -125,3 +125,21 @@ SKQ201920S,,,,GP03FLMB-RIS01-00-SIOENG000,7,1,2019-09-27T21:26:00,,CGMGP-03FLMB-
...
+SKQ202111S,,,,GP03FLMB-RIM01-02-CTDMOG068,8,1,2021-07-18T00:00:00,,CGMGP-03FLMB-00008,,CGINS-CTDMOG-13429,50.33,-144.39,,500,4145,,,
+SKQ202111S,,,,GP03FLMB-RIM01-02-CTDMOH069,8,1,2021-07-18T00:00:00,,CGMGP-03FLMB-00008,,CGINS-CTDMOH-13631,50.33,-144.39,,750,4145,,,
+SKQ202111S,,,,GP03FLMB-RIM01-02-CTDMOH070,8,1,2021-07-18T00:00:00,,CGMGP-03FLMB-00008,,CGINS-CTDMOH-12226,50.33,-144.39,,1000,4145,,,
+SKQ202111S,,,,GP03FLMB-RIM01-02-CTDMOH071,8,1,2021-07-18T00:00:00,,CGMGP-03FLMB-00008,,CGINS-CTDMOH-13632,50.33,-144.39,,1500,4145,,,
+SKQ202111S,,,,GP03FLMB-RIS01-00-SIOENG000,8,1,2021-07-18T00:00:00,,CGMGP-03FLMB-00008,,CGCON-SIOCON-14883,50.33,-144.39,,30,4145,,,
+SKQ202111S,,,,GP03FLMB-RIS01-03-DOSTAD000,8,1,2021-07-18T00:00:00,,CGMGP-03FLMB-00008,,CGINS-DOSTAD-00378,50.33,-144.39,,30,4145,,,
+SKQ202111S,,,,GP03FLMB-RIS01-04-PHSENF000,8,1,2021-07-18T00:00:00,,CGMGP-03FLMB-00008,,CGINS-PHSENF-P0144,50.33,-144.39,,30,4145,,,
+SKQ202111S,,,,GP03FLMB-RIS01-05-FLORTD000,8,1,2021-07-18T00:00:00,,CGMGP-03FLMB-00008,,CGINS-FLORTD-01324,50.33,-144.39,,30,4145,,,
```
Comment (asmith-whoi) [inline deployment/GP03FLMB_Deploy.csv:145]: SNs agree with config sheet, although one instrument may have been exchanged and not captured.

**Review by asmith-whoi — CHANGES_REQUESTED** [review]
See comments for FLMB. Should we change it? Or approve as is for now, and confirm/update the instrument replacement later, when we add the deployment details?

File: deployment/GP03FLMB_Deploy.csv (hunk 1 above)
Comment (rtravis-whoi) [reply deployment/GP03FLMB_Deploy.csv:145]: config and deploy csv updated:  swapped out P0144 for P0180

File: deployment/GP03FLMB_Deploy.csv [hunk 2]
```diff
@@ -125,3 +125,21 @@ SKQ201920S,,,,GP03FLMB-RIS01-00-SIOENG000,7,1,2019-09-27T21:26:00,,CGMGP-03FLMB-
...
+SKQ202111S,,,,GP03FLMB-RIM01-02-CTDMOG067,8,1,2021-07-18T00:00:00,,CGMGP-03FLMB-00008,,CGINS-CTDMOG-13431,50.33,-144.39,,350,4145,,,
+SKQ202111S,,,,GP03FLMB-RIM01-02-CTDMOG068,8,1,2021-07-18T00:00:00,,CGMGP-03FLMB-00008,,CGINS-CTDMOG-13429,50.33,-144.39,,500,4145,,,
+SKQ202111S,,,,GP03FLMB-RIM01-02-CTDMOH069,8,1,2021-07-18T00:00:00,,CGMGP-03FLMB-00008,,CGINS-CTDMOH-13631,50.33,-144.39,,750,4145,,,
+SKQ202111S,,,,GP03FLMB-RIM01-02-CTDMOH070,8,1,2021-07-18T00:00:00,,CGMGP-03FLMB-00008,,CGINS-CTDMOH-12226,50.33,-144.39,,1000,4145,,,
+SKQ202111S,,,,GP03FLMB-RIM01-02-CTDMOH071,8,1,2021-07-18T00:00:00,,CGMGP-03FLMB-00008,,CGINS-CTDMOH-13632,50.33,-144.39,,1500,4145,,,
+SKQ202111S,,,,GP03FLMB-RIS01-00-SIOENG000,8,1,2021-07-18T00:00:00,,CGMGP-03FLMB-00008,,CGCON-SIOCON-14883,50.33,-144.39,,30,4145,,,
+SKQ202111S,,,,GP03FLMB-RIS01-03-DOSTAD000,8,1,2021-07-18T00:00:00,,CGMGP-03FLMB-00008,,CGINS-DOSTAD-00378,50.33,-144.39,,30,4145,,,
+SKQ202111S,,,,GP03FLMB-RIS01-04-PHSENF000,8,1,2021-07-18T00:00:00,,CGMGP-03FLMB-00008,,CGINS-PHSENF-P0144,50.33,-144.39,,30,4145,,,
```
[the remark this answers is not in the document]
Comment (rtravis-whoi) [reply deployment/GP03FLMB_Deploy.csv]: updated!

**Review by asmith-whoi — APPROVED** [review]
Changes reviewed and approved

File: deployment/GP03FLMA_Deploy.csv [hunk 3]
```diff
@@ -125,3 +125,21 @@ SKQ201920S,,,,GP03FLMA-RIS01-00-SIOENG000,7,1,2019-09-29T04:40:00,,CGMGP-03FLMA-
...
+SKQ202111S,,,,GP03FLMA-RIM01-02-CTDMOG048,8,1,2021-07-18T00:00:00,,CGMGP-03FLMA-00008,,CGINS-CTDMOG-13598,49.97,-144.25,,500,4127,,,
+SKQ202111S,,,,GP03FLMA-RIM01-02-CTDMOH049,8,1,2021-07-18T00:00:00,,CGMGP-03FLMA-00008,,CGINS-CTDMOH-13449,49.97,-144.25,,750,4127,,,
+SKQ202111S,,,,GP03FLMA-RIM01-02-CTDMOH050,8,1,2021-07-18T00:00:00,,CGMGP-03FLMA-00008,,CGINS-CTDMOH-13447,49.97,-144.25,,1000,4127,,,
+SKQ202111S,,,,GP03FLMA-RIM01-02-CTDMOH051,8,1,2021-07-18T00:00:00,,CGMGP-03FLMA-00008,,CGINS-CTDMOH-13448,49.97,-144.25,,1500,4127,,,
+SKQ202111S,,,,GP03FLMA-RIS01-00-SIOENG000,8,1,2021-07-18T00:00:00,,CGMGP-03FLMA-00008,,CGCON-SIOCON-13998,49.97,-144.25,,30,4127,,,
+SKQ202111S,,,,GP03FLMA-RIS01-03-DOSTAD000,8,1,2021-07-18T00:00:00,,CGMGP-03FLMA-00008,,CGINS-DOSTAD-00377,49.97,-144.25,,30,4127,,,
+SKQ202111S,,,,GP03FLMA-RIS01-04-PHSENF000,8,1,2021-07-18T00:00:00,,CGMGP-03FLMA-00008,,CGINS-PHSENF-P0142,49.97,-144.25,,30,4127,,,
+SKQ202111S,,,,GP03FLMA-RIS01-05-FLORTD000,8,1,2021-07-18T00:00:00,,CGMGP-03FLMA-00008,,CGINS-FLORTD-01323,49.97,-144.25,,30,4127,,,
```
Comment (cdobs) [inline deployment/GP03FLMA_Deploy.csv:145]: Reviewed configuration, looks good

File: deployment/GP03FLMB_Deploy.csv (hunk 1 above)
Comment (cdobs) [reply deployment/GP03FLMB_Deploy.csv:145]: Reviewed configuration, looks good. Swap of P0144 for P0180 looks good.

File: deployment/GP02HYPM_Deploy.csv [hunk 4]
```diff
@@ -97,3 +91,17 @@ SKQ201920S,,,,GP02HYPM-WFP03-01-FLORDL000,7,1,2019-09-26T00:05:00,,CGMGP-02HYPM-
...
+SKQ202111S,,,,GP02HYPM-WFP02-01-FLORDL000,8,1,2021-07-18T00:00:00,,CGMGP-02HYPM-00008,CGVEH-GWFPGX-10402,CGINS-FLORDL-03283,50.07,-144.8,,2090,4219,,,
+SKQ202111S,,,,GP02HYPM-WFP02-03-DOSTAL000,8,1,2021-07-18T00:00:00,,CGMGP-02HYPM-00008,CGVEH-GWFPGX-10402,CGINS-DOSTAL-01480,50.07,-144.8,,2090,4219,,,
+SKQ202111S,,,,GP02HYPM-WFP02-04-CTDPFL000,8,1,2021-07-18T00:00:00,,CGMGP-02HYPM-00008,CGVEH-GWFPGX-10402,CGINS-CTDPFL-00134,50.07,-144.8,,2090,4219,,,
+SKQ202111S,,,,GP02HYPM-WFP02-05-VEL3DL000,8,1,2021-07-18T00:00:00,,CGMGP-02HYPM-00008,CGVEH-GWFPGX-10402,CGINS-VEL3DL-01108,50.07,-144.8,,2090,4219,,,
+SKQ202111S,,,,GP02HYPM-WFP03-00-WFPENG000,8,1,2021-07-18T00:00:00,,CGMGP-02HYPM-00008,CGVEH-GWFPGX-93602,CGCON-GWFPGX-93602,50.07,-144.8,,4063,4219,,,
+SKQ202111S,,,,GP02HYPM-WFP03-01-FLORDL000,8,1,2021-07-18T00:00:00,,CGMGP-02HYPM-00008,CGVEH-GWFPGX-93602,CGINS-FLORDL-02736,50.07,-144.8,,4063,4219,,,
+SKQ202111S,,,,GP02HYPM-WFP03-03-DOSTAL000,8,1,2021-07-18T00:00:00,,CGMGP-02HYPM-00008,CGVEH-GWFPGX-93602,CGINS-DOSTAL-01086,50.07,-144.8,,4063,4219,,,
```
Comment (cdobs) [inline deployment/GP02HYPM_Deploy.csv:105]: Based on the cals in 12936-02 on Vault, shouldn't this be CGINS-DOSTAL-01086? Or have we just not received/posted the new cals for this WFP yet?

File: deployment/GP02HYPM_Deploy.csv [hunk 5]
```diff
@@ -39,8 +35,6 @@ TN323,,RB1605,,GP02HYPM-WFP03-01-FLORDL000,3,1,2015-06-04T23:38:00,2016-07-01T17
 TN323,,RB1605,,GP02HYPM-WFP03-03-DOSTAL000,3,1,2015-06-04T23:38:00,2016-07-01T17:57:00,CGMGP-02HYPM-00003,CGVEH-GWFPGX-10403,CGINS-DOSTAL-01478,50.07983,-144.803,,4070,4219,,,
 TN323,,RB1605,,GP02HYPM-WFP03-04-CTDPFL000,3,1,2015-06-04T23:38:00,2016-07-01T17:57:00,CGMGP-02HYPM-00003,CGVEH-GWFPGX-10403,CGINS-CTDPFL-00132,50.07983,-144.803,,4070,4219,,,
 TN323,,RB1605,,GP02HYPM-WFP03-05-VEL3DL000,3,1,2015-06-04T23:38:00,2016-07-01T17:57:00,CGMGP-02HYPM-00003,CGVEH-GWFPGX-10403,CGINS-VEL3DL-01115,50.07983,-144.803,,4070,4219,,,
-#TN323,,RB1605,,GP02HYPM-MPM01-02-ZPLSGA009,3,1,2015-06-04T23:38:00,2016-07-01T17:57:00,CGMGP-02HYPM-00003,,,50.07983,-144.803,,,4219,Not Deployed,,
-#TN323,,RB1605,,GP02HYPM-MPM01-02-ZPLSGA010,3,1,2015-06-04T23:38:00,2016-07-01T17:57:00,CGMGP-02HYPM-00003,,,50.07983,-144.803,,,4219,Not Deployed,,
```
Comment (cdobs) [inline deployment/GP02HYPM_Deploy.csv:43]: Looks good

File: deployment/GP02HYPM_Deploy.csv [hunk 6]
```diff
@@ -25,8 +23,6 @@ MV1404,,TN323,,GP02HYPM-WFP03-01-FLORDL000,2,1,2014-06-22T02:14:00,2015-06-02T00
 MV1404,,TN323,,GP02HYPM-WFP03-03-DOSTAL000,2,1,2014-06-22T02:14:00,2015-06-02T00:00:00,CGMGP-02HYPM-00002,CGVEH-GWFPGX-10402,CGINS-DOSTAL-01480,50.07833,-144.80533,,4038,4219,,,
 MV1404,,TN323,,GP02HYPM-WFP03-04-CTDPFL000,2,1,2014-06-22T02:14:00,2015-06-02T00:00:00,CGMGP-02HYPM-00002,CGVEH-GWFPGX-10402,CGINS-CTDPFL-00134,50.07833,-144.80533,,4038,4219,,,
 MV1404,,TN323,,GP02HYPM-WFP03-05-VEL3DL000,2,1,2014-06-22T02:14:00,2015-06-02T00:00:00,CGMGP-02HYPM-00002,CGVEH-GWFPGX-10402,CGINS-VEL3DL-13551,50.07833,-144.80533,,4038,4219,,,
-#MV1404,,TN323,,GP02HYPM-MPM01-02-ZPLSGA009,2,1,2014-06-22T02:14:00,2015-06-02T00:00:00,CGMGP-02HYPM-00002,,,50.07833,-144.80533,,,4219,Not Deployed,,
-#MV1404,,TN323,,GP02HYPM-MPM01-02-ZPLSGA010,2,1,2014-06-22T02:14:00,2015-06-02T00:00:00,CGMGP-02HYPM-00002,,,50.07833,-144.80533,,,4219,Not Deployed,,
```
Comment (cdobs) [inline deployment/GP02HYPM_Deploy.csv:29]: Looks good

File: deployment/GP02HYPM_Deploy.csv [hunk 7]
```diff
@@ -11,8 +11,6 @@ MV1309,,MV1404,,GP02HYPM-WFP03-01-FLORDL000,1,1,2013-07-26T08:11:00,2014-06-14T0
 MV1309,,MV1404,,GP02HYPM-WFP03-03-DOSTAL000,1,1,2013-07-26T08:11:00,2014-06-14T00:00:00,CGMGP-02HYPM-00001,CGVEH-GWFPGX-77401,CGINS-DOSTAL-01106,50.07067,-144.798,,4061,4219,,,
 MV1309,,MV1404,,GP02HYPM-WFP03-04-CTDPFL000,1,1,2013-07-26T08:11:00,2014-06-14T00:00:00,CGMGP-02HYPM-00001,CGVEH-GWFPGX-77401,CGINS-CTDPFL-00097,50.07067,-144.798,,4061,4219,,,
 MV1309,,MV1404,,GP02HYPM-WFP03-05-VEL3DL000,1,1,2013-07-26T08:11:00,2014-06-14T00:00:00,CGMGP-02HYPM-00001,CGVEH-GWFPGX-77401,CGINS-VEL3DL-01025,50.07067,-144.798,,4061,4219,,,
-#MV1309,,MV1404,,GP02HYPM-MPM01-02-ZPLSGA009,1,1,2013-07-26T08:11:00,2014-06-14T00:00:00,CGMGP-02HYPM-00001,,,50.07067,-144.798,,,4219,Not Deployed,,
-#MV1309,,MV1404,,GP02HYPM-MPM01-02-ZPLSGA010,1,1,2013-07-26T08:11:00,2014-06-14T00:00:00,CGMGP-02HYPM-00001,,,50.07067,-144.798,,,4219,Not Deployed,,
```
Comment (cdobs) [inline deployment/GP02HYPM_Deploy.csv:15]: Looks good

**Review by cdobs — COMMENTED** [review]
Mostly everything looks good, just had one question about HYPM DOSTA

File: deployment/GP02HYPM_Deploy.csv (hunk 4 above)
Comment (rtravis-whoi) [reply deployment/GP02HYPM_Deploy.csv]: Yes - you are correct!  This has been updated.  Need to remember to add this sensor to sensor bulk load.

Comment (rtravis-whoi) [reply deployment/GP02HYPM_Deploy.csv:105]: After further investigation we believe 1086 is correct and that the vendor calibration document is incorrect.  Will reach out to Aanderaa to confirm & get a corrected vendor cal doc.  Will also visually confirm SN once things are unloaded in Seward.

Comment (cdobs) [reply deployment/GP02HYPM_Deploy.csv:105]: Looks good

**Review by cdobs — APPROVED** [review]

File: deployment/GP03FLMB_Deploy.csv (hunk 1 above)
Comment (rtravis-whoi) [reply deployment/GP03FLMB_Deploy.csv:145]: Updated D00001 controllers - they were reversed.  MSIOC changed to 12371; SSIOC changed to 12365.

File: deployment/GP03FLMA_Deploy.csv [hunk 8]
```diff
@@ -17,7 +17,7 @@ MV1309,,MV1404,,GP03FLMA-RIS01-00-SIOENG000,1,1,2013-07-21T22:44:00,2014-06-15T1
 MV1309,,MV1404,,GP03FLMA-RIS01-03-DOSTAD000,1,1,2013-07-21T22:44:00,2014-06-15T17:30:00,CGMGP-03FLMA-00001,,CGINS-DOSTAD-00129,49.9795,-144.254,,28,4127,,,
 MV1309,,MV1404,,GP03FLMA-RIS01-04-PHSENF000,1,1,2013-07-21T22:44:00,2014-06-15T17:30:00,CGMGP-03FLMA-00001,,CGINS-PHSENF-P0080,49.9795,-144.254,,28,4127,,,
 MV1309,,MV1404,,GP03FLMA-RIS01-05-FLORTD000,1,1,2013-07-21T22:44:00,2014-06-15T17:30:00,CGMGP-03FLMA-00001,,CGINS-FLORTD-01005,49.9795,-144.254,,28,4127,,,
-MV1404,,TN323,,GP03FLMA-RIM01-00-SIOENG000,2,1,2014-06-18T05:56:00,2015-06-05T18:01:00,CGMGP-03FLMA-00002,,CGCON-SIOCON-12378,49.9775,-144.2463,,1506,4127,,,
+MV1404,,TN323,,GP03FLMA-RIM01-00-SIOENG000,2,1,2014-06-18T05:56:00,2015-06-05T18:01:00,CGMGP-03FLMA-00002,,CGCON-SIOCON-13278,49.9775,-144.2463,,1506,4127,,,
```
Comment (rtravis-whoi) [inline deployment/GP03FLMA_Deploy.csv:20]: Updated SSIOC SN from 12378 to 13278.

**Comment by rtravis-whoi** [comment]
2 updates that need review:
GP03FLMA - D00002 SSIOC SN was incorrect - (see cgsn data slack channel) - changed from 12378 to 13278
GP03FLMB - D00001 MSIOC and SSIOC were reversed - MSIOC changed to 12371 and SSIOC changed to 12365.

**Comment by rtravis-whoi** [comment]
Added pre-cruise Papa glider deploy csvs:
GP05MOAS-GL276_Deploy
GP05MOAS-GL363_Deploy

File: deployment/GP03FLMA_Deploy.csv (hunk 8 above)
Comment (rtravis-whoi) [reply deployment/GP03FLMA_Deploy.csv:20]: Updated D00001 MSIOC and SSIOC (they were reversed)

**Comment by rtravis-whoi** [comment]
Final controllers update for Papa:
GP03FLMA D00001 - MSIOC and SSIOC were reversed per James' controller tracking 6-10-21 Ss (posted on slack channel)

File: deployment/GP03FLMA_Deploy.csv [hunk 9]
```diff
@@ -1,5 +1,5 @@
 CUID_Deploy,deployedBy,CUID_Recover,recoveredBy,Reference Designator,deploymentNumber,versionNumber,startDateTime,stopDateTime,mooring.uid,node.uid,sensor.uid,lat,lon,orbit,deployment_depth,water_depth,notes,electrical.uid,assembly_template_revision
-MV1309,,MV1404,,GP03FLMA-RIM01-00-SIOENG000,1,1,2013-07-21T22:44:00,2014-06-15T17:30:00,CGMGP-03FLMA-00001,,CGCON-SIOCON-11690,49.9795,-144.254,,1506,4127,,,
+MV1309,,MV1404,,GP03FLMA-RIM01-00-SIOENG000,1,1,2013-07-21T22:44:00,2014-06-15T17:30:00,CGMGP-03FLMA-00001,,CGCON-SIOCON-12372,49.9795,-144.254,,1506,4127,,,
```
Comment (cdobs) [inline deployment/GP03FLMA_Deploy.csv:2]: Looks good

File: deployment/GP03FLMA_Deploy.csv [hunk 10]
```diff
@@ -13,11 +13,11 @@ MV1309,,MV1404,,GP03FLMA-RIM01-02-CTDMOG048,1,1,2013-07-21T22:44:00,2014-06-15T1
 MV1309,,MV1404,,GP03FLMA-RIM01-02-CTDMOH049,1,1,2013-07-21T22:44:00,2014-06-15T17:30:00,CGMGP-03FLMA-00001,,CGINS-CTDMOH-10226,49.9795,-144.254,,748,4127,,,
 MV1309,,MV1404,,GP03FLMA-RIM01-02-CTDMOH050,1,1,2013-07-21T22:44:00,2014-06-15T17:30:00,CGMGP-03FLMA-00001,,CGINS-CTDMOH-10229,49.9795,-144.254,,998,4127,,,
 MV1309,,MV1404,,GP03FLMA-RIM01-02-CTDMOH051,1,1,2013-07-21T22:44:00,2014-06-15T17:30:00,CGMGP-03FLMA-00001,,CGINS-CTDMOH-10267,49.9795,-144.254,,1498,4127,,,
-MV1309,,MV1404,,GP03FLMA-RIS01-00-SIOENG000,1,1,2013-07-21T22:44:00,2014-06-15T17:30:00,CGMGP-03FLMA-00001,,CGCON-SIOCON-12372,49.9795,-144.254,,28,4127,,,
+MV1309,,MV1404,,GP03FLMA-RIS01-00-SIOENG000,1,1,2013-07-21T22:44:00,2014-06-15T17:30:00,CGMGP-03FLMA-00001,,CGCON-SIOCON-11690,49.9795,-144.254,,28,4127,,,
```
Comment (cdobs) [inline deployment/GP03FLMA_Deploy.csv:16]: Looks good

File: deployment/GP03FLMA_Deploy.csv [hunk 11]
```diff
@@ -13,11 +13,11 @@ MV1309,,MV1404,,GP03FLMA-RIM01-02-CTDMOG048,1,1,2013-07-21T22:44:00,2014-06-15T1
...
 MV1309,,MV1404,,GP03FLMA-RIS01-03-DOSTAD000,1,1,2013-07-21T22:44:00,2014-06-15T17:30:00,CGMGP-03FLMA-00001,,CGINS-DOSTAD-00129,49.9795,-144.254,,28,4127,,,
 MV1309,,MV1404,,GP03FLMA-RIS01-04-PHSENF000,1,1,2013-07-21T22:44:00,2014-06-15T17:30:00,CGMGP-03FLMA-00001,,CGINS-PHSENF-P0080,49.9795,-144.254,,28,4127,,,
 MV1309,,MV1404,,GP03FLMA-RIS01-05-FLORTD000,1,1,2013-07-21T22:44:00,2014-06-15T17:30:00,CGMGP-03FLMA-00001,,CGINS-FLORTD-01005,49.9795,-144.254,,28,4127,,,
-MV1404,,TN323,,GP03FLMA-RIM01-00-SIOENG000,2,1,2014-06-18T05:56:00,2015-06-05T18:01:00,CGMGP-03FLMA-00002,,CGCON-SIOCON-12378,49.9775,-144.2463,,1506,4127,,,
+MV1404,,TN323,,GP03FLMA-RIM01-00-SIOENG000,2,1,2014-06-18T05:56:00,2015-06-05T18:01:00,CGMGP-03FLMA-00002,,CGCON-SIOCON-13278,49.9775,-144.2463,,1506,4127,,,
```
Comment (cdobs) [inline deployment/GP03FLMA_Deploy.csv:20]: Looks good

File: deployment/GP03FLMB_Deploy.csv [hunk 12]
```diff
@@ -1,5 +1,5 @@
 CUID_Deploy,deployedBy,CUID_Recover,recoveredBy,Reference Designator,deploymentNumber,versionNumber,startDateTime,stopDateTime,mooring.uid,node.uid,sensor.uid,lat,lon,orbit,deployment_depth,water_depth,notes,electrical.uid,assembly_template_revision
-MV1309,,MV1404,,GP03FLMB-RIM01-00-SIOENG000,1,1,2013-07-24T06:32:00,2014-06-18T14:00:00,CGMGP-03FLMB-00001,,CGCON-SIOCON-12365,50.33167,-144.40083,,1507,4145,,,
+MV1309,,MV1404,,GP03FLMB-RIM01-00-SIOENG000,1,1,2013-07-24T06:32:00,2014-06-18T14:00:00,CGMGP-03FLMB-00001,,CGCON-SIOCON-12371,50.33167,-144.40083,,1507,4145,,,
```
Comment (cdobs) [inline deployment/GP03FLMB_Deploy.csv:2]: Looks good

File: deployment/GP03FLMB_Deploy.csv [hunk 13]
```diff
@@ -13,7 +13,7 @@ MV1309,,MV1404,,GP03FLMB-RIM01-02-CTDMOG068,1,1,2013-07-24T06:32:00,2014-06-18T1
 MV1309,,MV1404,,GP03FLMB-RIM01-02-CTDMOH069,1,1,2013-07-24T06:32:00,2014-06-18T14:00:00,CGMGP-03FLMB-00001,,CGINS-CTDMOH-10266,50.33167,-144.40083,,750,4145,,,
 MV1309,,MV1404,,GP03FLMB-RIM01-02-CTDMOH070,1,1,2013-07-24T06:32:00,2014-06-18T14:00:00,CGMGP-03FLMB-00001,,CGINS-CTDMOH-10265,50.33167,-144.40083,,1000,4145,,,
 MV1309,,MV1404,,GP03FLMB-RIM01-02-CTDMOH071,1,1,2013-07-24T06:32:00,2014-06-18T14:00:00,CGMGP-03FLMB-00001,,CGINS-CTDMOH-10264,50.33167,-144.40083,,1500,4145,,,
-MV1309,,MV1404,,GP03FLMB-RIS01-00-SIOENG000,1,1,2013-07-24T06:32:00,2014-06-18T14:00:00,CGMGP-03FLMB-00001,,CGCON-SIOCON-12371,50.33167,-144.40083,,30,4145,,,
+MV1309,,MV1404,,GP03FLMB-RIS01-00-SIOENG000,1,1,2013-07-24T06:32:00,2014-06-18T14:00:00,CGMGP-03FLMB-00001,,CGCON-SIOCON-12365,50.33167,-144.40083,,30,4145,,,
```
Comment (cdobs) [inline deployment/GP03FLMB_Deploy.csv:16]: Looks good

File: deployment/GP05MOAS-GL276_Deploy.csv [hunk 14]
```diff
@@ -7,3 +7,7 @@ SR1811,,TY-1902,,GP05MOAS-GL276-00-ENG000000,2,1,2018-07-23T06:33:00,2019-02-12T
 SR1811,,TY-1902,,GP05MOAS-GL276-01-FLORDM000,2,1,2018-07-23T06:33:00,2019-02-12T21:15:00,CGVGP-05MOAS-00000,CGVEH-GLDROO-00276,CGINS-FLORTM-03260,50.1295,-144.5424,,1000,1000,FLORT deployed in place of customary FLORD,,
 SR1811,,TY-1902,,GP05MOAS-GL276-02-DOSTAM000,2,1,2018-07-23T06:33:00,2019-02-12T21:15:00,CGVGP-05MOAS-00000,CGVEH-GLDROO-00276,CGINS-DOSTAM-00145,50.1295,-144.5424,,1000,1000,,,
 SR1811,,TY-1902,,GP05MOAS-GL276-04-CTDGVM000,2,1,2018-07-23T06:33:00,2019-02-12T21:15:00,CGVGP-05MOAS-00000,CGVEH-GLDROO-00276,CGINS-CTDGVM-00104,50.1295,-144.5424,,1000,1000,,,
+SKQ202111S,,,,GP05MOAS-GL276-00-ENG000000,3,1,2021-07-18T00:00:00,,CGVGP-05MOAS-00000,CGVEH-GLDROO-00276,CGCON-GLDROO-00276,50.12,-144.54,,1000,1000,,,
+SKQ202111S,,,,GP05MOAS-GL276-01-FLORDM000,3,1,2021-07-18T00:00:00,,CGVGP-05MOAS-00000,CGVEH-GLDROO-00276,CGINS-FLORDM-03583,50.12,-144.54,,1000,1000,,,
+SKQ202111S,,,,GP05MOAS-GL276-02-DOSTAM000,3,1,2021-07-18T00:00:00,,CGVGP-05MOAS-00000,CGVEH-GLDROO-00276,CGINS-DOSTAM-00398,50.12,-144.54,,1000,1000,,,
+SKQ202111S,,,,GP05MOAS-GL276-04-CTDGVM000,3,1,2021-07-18T00:00:00,,CGVGP-05MOAS-00000,CGVEH-GLDROO-00276,CGINS-CTDGVM-09238,50.12,-144.54,,1000,1000,,,
```
Comment (cdobs) [inline deployment/GP05MOAS-GL276_Deploy.csv:13]: Looks good

File: deployment/GP05MOAS-GL363_Deploy.csv [hunk 15]
```diff
@@ -11,3 +11,7 @@ ZP20200930,,ZP20200930,,GP05MOAS-GL363-00-ENG000000,4,1,2020-10-02T12:59:00,2020
 ZP20200930,,ZP20200930,,GP05MOAS-GL363-01-FLORDM000,4,1,2020-10-02T12:59:00,2020-10-02T19:50:00,CGVGP-05MOAS-00000,CGVEH-GLDROO-00363,CGINS-FLORDM-03063,46.01577,-133.00577,,1000,1000,,,
 ZP20200930,,ZP20200930,,GP05MOAS-GL363-02-DOSTAM000,4,1,2020-10-02T12:59:00,2020-10-02T19:50:00,CGVGP-05MOAS-00000,CGVEH-GLDROO-00363,CGINS-DOSTAM-00341,46.01577,-133.00577,,1000,1000,,,
 ZP20200930,,ZP20200930,,GP05MOAS-GL363-04-CTDGVM000,4,1,2020-10-02T12:59:00,2020-10-02T19:50:00,CGVGP-05MOAS-00000,CGVEH-GLDROO-00363,CGINS-CTDGVM-00104,46.01577,-133.00577,,1000,1000,,,
+SKQ202111S,,,,GP05MOAS-GL363-00-ENG000000,5,1,2021-07-18T00:00:00,,CGVGP-05MOAS-00000,CGVEH-GLDROO-00363,CGCON-GLDROO-00363,46.01,-133,,1000,1000,,,
+SKQ202111S,,,,GP05MOAS-GL363-01-FLORDM000,5,1,2021-07-18T00:00:00,,CGVGP-05MOAS-00000,CGVEH-GLDROO-00363,CGINS-FLORDM-03063,46.01,-133,,1000,1000,,,
+SKQ202111S,,,,GP05MOAS-GL363-02-DOSTAM000,5,1,2021-07-18T00:00:00,,CGVGP-05MOAS-00000,CGVEH-GLDROO-00363,CGINS-DOSTAM-00341,46.01,-133,,1000,1000,,,
+SKQ202111S,,,,GP05MOAS-GL363-04-CTDGVM000,5,1,2021-07-18T00:00:00,,CGVGP-05MOAS-00000,CGVEH-GLDROO-00363,CGINS-CTDGVM-00104,46.01,-133,,1000,1000,,,
```
Comment (cdobs) [inline deployment/GP05MOAS-GL363_Deploy.csv:17]: Looks good

**Review by cdobs — APPROVED** [review]
Reviewed glider deploy csvs and updates to persistor SNs on flanking moorings. Had previously reviewed deployment configuration for upcoming Papa turn.

File: deployment/GP03FLMA_Deploy.csv (hunk 9 above)
Comment (asmith-whoi) [inline deployment/GP03FLMA_Deploy.csv:2]: OK

File: deployment/GP03FLMB_Deploy.csv (hunk 12 above)
Comment (asmith-whoi) [inline deployment/GP03FLMB_Deploy.csv:2]: OK

File: deployment/GP03FLMB_Deploy.csv (hunk 13 above)
Comment (asmith-whoi) [inline deployment/GP03FLMB_Deploy.csv:16]: OK

File: deployment/GP05MOAS-GL276_Deploy.csv (hunk 14 above)
Comment (asmith-whoi) [inline deployment/GP05MOAS-GL276_Deploy.csv:13]: OK

File: deployment/GP05MOAS-GL363_Deploy.csv (hunk 15 above)
Comment (asmith-whoi) [inline deployment/GP05MOAS-GL363_Deploy.csv:17]: OK

**Pull request merged**